packages/vm/src/runCode.ts

```diff
@@ -74,23 +74,24 @@ export default function runCode(this: VM, opts: RunCodeOpts): Promise<ExecResult
   const block = opts.block ?? Block.fromBlockData({}, { common: this._common })
 
   // Backwards compatibility
```

Comment:
Since it doesn't look like we pass `opts.txContext` into `runCode` anywhere (in our source or tests), I say we can remove the opt and this note

Reply:
nvm I will handle this in my own develop breaking changes PR